.gitlab-ci.yml

```diff
@@ -0,0 +1,66 @@
+# Keep this file in sync with docker-compose.yml.
+
+# Build JAVA applications using Apache Maven (http://maven.apache.org)
+# For docker image tags see https://hub.docker.com/_/maven/
+#
+# For general lifecycle information see https://maven.apache.org/guides/introduction/introduction-to-the-lifecycle.html
+
+image: dockerhub.ebi.ac.uk/ega/ega-data-api-build-container:latest
+
+# Add the service of docker in docker to allow us to compile the docker images
+# https://docs.gitlab.com/runner/executors/kubernetes.html#using-dockerdind
+services:
+  - name: docker:dind
+    alias: dataedge
+
+## List of jobs:
+stages:
+  - unitTest
+  - integrationTest
+  - imageBuild
+
+# Cache downloaded dependencies and plugins between builds.
+# To keep cache across branches add 'key: "$CI_JOB_NAME"'
+cache:
+  paths:
+    - .m2/repository
+
+variables:
+  MAVEN_OPTS: "-Dmaven.repo.local=$CI_PROJECT_DIR/.m2/repository"
+  TRAVIS_BUILD_DIR: $CI_PROJECT_DIR
```

Comment:
Do we need this variable if the .travis.yml was deleted?

Reply:
We do need it @bela-juhasz but I've tell you why, this is because the variable in the environment is called like that so that's the reason why I used it (I was a bit afraid of rename it to be honest 👍 )